spec/system/visit_user_journey_view_certificate_page_spec.rb

```diff
@@ -138,4 +138,40 @@
       expect(page).to have_content(t('user_journey.certificate.stop_using_secondary_warning'))
     end
   end
+
+  context 'Stop using certificate' do
+    it 'is secondary' do
+      component = create(:sp_component, team_id: user.team)
+      primary = create(:msa_signing_certificate, component: component)
+      secondary = create(:msa_signing_certificate, component: component)
```

Comment:
```
component = create(:sp_component, team_id: user.team) #should this be :msa_component
primary = create(:msa_signing_certificate, component: component)
```
also
primary and secondary not used; I understand why you have named them. how about 
something like this
```
    before(:example) do
      let(:msa_component_with_primary_and_secondary_certificate) do
        create(:sp_component, team_id: user.team)
        create(:msa_signing_certificate, component: component)
        create(:msa_signing_certificate, component: component)
      end
    end
    it 'is secondary' do
      msa_component_with_primary_and_secondary_certificate
      travel_to Time.now + 11.minutes

      visit root_path

      click_link 'Signing certificate (secondary)'
      click_link('Stop using this certificate')
      expect(page).to have_selector('h1', text: 'Manage certificates')
      expect(page).not_to have_content('Signing certificate (secondary)')
      expect(page).to have_content('Signing certificate')
    end
```

Reply:
thanks @jkunle :) I used your idea except I made a method and called it from the `before(:example)` block, otherwise the `let` vars can't be accessed from inside the `it` blocks annoyingly :(

Reply:
This is satisfactory